docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -185,29 +185,77 @@ unavailable occurs with probability that the birth is in a BEmONC facility times
 
 When we fill in the location-specific values for delivery facility rates, azithromycin coverage, relative risk of maternal sepsis incidence with azithromycin, 
 and maternal sepsis incidence probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{maternal_sepsis}|\text{azithromycin})` 
-and :math:`p(\text{maternal_sepsis}|\text{no azithromycin})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
+and :math:`p(\text{maternal_sepsis}|\text{no azithromycin})`), , which we can solve analytically.
 
-**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that
-:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+As mentioned above, it is convenient to model this intervention like a dichotomous risk factor, so that we can reuse the
+:class:`Risk<vivarium_public_health.risks.base_risk.Risk>`
+and :class:`RiskEffect<vivarium_public_health.risks.effect.RiskEffect>` components in Vivarium Public Health,
+rather than having to write new components from scratch.
+Calling the intervention a risk factor can sound a bit confusing because intervention access is a good thing, so it doesn't sound "risky."
```

Comment:
I am starting to think that flipping the framing is also confusing, and it might be less confusing to have negative PAFs.  No change needed here, but let's try the negative-PAF version in the future and see if that is clearer.

Reply:
Great! That is my typical workflow and if I am remembering correctly that is what we had decided in our group discussion that came out of [this thread](https://ihme.slack.com/archives/C221EE33J/p1751928058522109?thread_ts=1751580075.153659&cid=C221EE33J)